src/style.css

```diff
@@ -1,3 +1,85 @@
-body {
+html,
+body,
+header,
+nav,
+div,
+ul {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  --theme-color: #00acdc;
+}
+
+body {
+  line-height: 1;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav {
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+}
+
+.nav__list-item {
+  position: relative;
+  display: inline-block;
+  list-style: none;
+  margin-left: 20px;
+}
+
+.nav__list-item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: block;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: inherit;
+  height: 60px;
+  line-height: 60px;
+}
+
+.container {
+  max-width: 1200px;
+}
+
+.nav__list li:hover {
+  color: var(--theme-color);
+}
+
+.is-active {
+  color: var(--theme-color);
+}
+
+.is-active:after {
+  content: "";
+  height: 4px;
+  width: 100%;
+  border-radius: 8px;
```

Comment:
border-radius should be half of width/height. If you set bigger value it is not a good practice.